src/config.js

```diff
@@ -50,4 +52,6 @@ module.exports = {
   mattermostInvitationLink: process.env.MATTERMOST_INVITATION_LINK || '',
   investigationReportsIframeURL: process.env.INVESTIGATION_REPORTS_IFRAME_URL || '',
   leavesEmail: process.env.LEAVES_EMAIL || 'depart@beta.gouv.fr',
+  // staging tests values
```

Comment:
Autre suggestion, suite aux guidelines sur _[contributing](https://github.com/betagouv/secretariat/blob/master/CONTRIBUTING.md#configuration)_, pourrait-on rendre la config optionnelle ? Si je lance les cron en local sans avoir les envvars _githubOrganizationName_ et _githubOrgAdminToken_ ça me lance une erreur (ce qui est bien sur normal) : 

```
(node:90935) UnhandledPromiseRejectionWarning: Error: HttpError: Requires authentication
```
Et si la fonction `addGithubUserToOrganization` ne faisait pas l'appel en absence de ces variables ? ça pourrait logger qqchose comme "_Unable to launch bla bla without env vars githubOrganizationName and githubOrgAdminToken_".

Ça pourrait se faire dans un deuxième temps aussi, ou bien on peut se dire que ces env-vars sont un must-have.